core-java/src/main/java/org/spine3/server/internal/CommandHandlerMethod.java

```diff
@@ -138,58 +135,47 @@ protected static boolean acceptsCorrectParams(Method method) {
      */
     @CheckReturnValue
     public static Map<CommandClass, CommandHandlerMethod> scan(Object object) {
```

Comment:
I'd add to docs something like this:
''
@param object the object implementing {@link CommandHandlingObject} that keeps command handler methods
@see CommandHandlingObject
''
So that client doesn't have to look into the method implementation.
Other than that, LGTM.